### PR TITLE
Feat/us debt

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: client/app/account/billing/history/balance/translations/Messages_en_GB.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+    <translation id="billing_history_balance_title">Pay my balance</translation>
+    <translation id="billing_history_balance_cancel">Cancel</translation>
+    <translation id="billing_history_balance_explain">Your balance is currently {{ balance }}.</translation>
+    <translation id="billing_history_balance_no_balance">No balance has been found on your account. Please <a data-ui-sref="{{ backState }}">click here</a> to go back to your bill list.</translation>
+    <translation id="billing_history_balance_choose_payment_method">Choose the payment method</translation>
+    <translation id="billing_history_balance_add_payment_method">Add a payment method</translation>
+    <translation id="billing_history_balance_payment_doing">A balance payment is already in progress. Please wait that this paiement is done before paying a new balance.</translation>
```
Comment (antleblanc) [inline client/app/account/billing/history/balance/translations/Messages_en_GB.xml]: `paiement` => `payment`?

File: client/app/account/billing/history/balance/translations/Messages_en_US.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+    <translation id="billing_history_balance_title">Pay my balance</translation>
+    <translation id="billing_history_balance_cancel">Cancel</translation>
+    <translation id="billing_history_balance_explain">Your balance is currently {{ balance }}.</translation>
+    <translation id="billing_history_balance_no_balance">No balance has been found on your account. Please <a data-ui-sref="{{ backState }}">click here</a> to go back to your bill list.</translation>
+    <translation id="billing_history_balance_choose_payment_method">Choose the payment method</translation>
+    <translation id="billing_history_balance_add_payment_method">Add a payment method</translation>
+    <translation id="billing_history_balance_no_payment_method">You don't have valid payment method to pay your balance. Please <a data-ui-sref="{{ addPaymentMethodState }}">add a valid payment method</a> before pay it.</translation>
+    <translation id="billing_history_balance_payment_doing">A balance payment is already in progress. Please wait that this paiement is done before paying a new balance.</translation>
```
Comment (antleblanc) [inline client/app/account/billing/history/balance/translations/Messages_en_US.xml]: `paiement` => `payment`?

File: client/app/account/billing/history/balance/translations/Messages_fr_FR.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+    <translation id="billing_history_balance_pay_title">Règlement de mon solde</translation>
+    <translation id="billing_history_balance_cancel">Annuler</translation>
+    <translation id="billing_history_balance_explain">Votre solde est actuellement de {{ balance }}.</translation>
+    <translation id="billing_history_balance_no_balance">Aucune balance n'a été trouvée sur votre compte. <a data-ui-sref="{{ backState }}">Cliquez ici</a> pour retourner à la liste des factures.</translation>
+    <translation id="billing_history_balance_choose_payment_method">Choisissez une méthode de paiement</translation>
+    <translation id="billing_history_balance_add_payment_method">Ajoutez une méthode de paiement</translation>
+    <translation id="billing_history_balance_no_payment_method">Vous n'avez aucune méthode de paiement valide. <a data-ui-sref="{{ addPaymentMethodState }}">Ajoutez une méthode de paiement valide</a> avant régler votre solde.</translation>
```
Comment (antleblanc) [inline client/app/account/billing/history/balance/translations/Messages_fr_FR.xml]: `avant régler votre solde` => `avant de régler votre solde`.

File: client/app/account/billing/history/balance/translations/Messages_fr_FR.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <translation id="billing_history_balance_payment_doing">Un règlement de solde est déjà en cours. Attendez que ce paiement soit effectuer avant de règler un nouveau solde.</translation>
```
Comment (antleblanc) [inline client/app/account/billing/history/balance/translations/Messages_fr_FR.xml]: `effectuer` => `effectué`.

File: client/app/account/billing/history/balance/translations/Messages_fr_FR.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <translation id="billing_history_balance_payment_go_tracking">Rendez-vous <a data-ui-sref="{{ trackState }}">sur la page des suivis de paiement</a> pour plus d'informations.</translation>
+
+    <translation id="billing_history_balance_load_error">Un erreur est survenue lors du chargement de votre solde.</translation>
```
Comment (antleblanc) [inline client/app/account/billing/history/balance/translations/Messages_fr_FR.xml]: `Un` => `Une`.

File: client/app/account/billing/payments/request/billing-payments-request.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    <div class="text-center"
+         data-ng-if="$ctrl.loading.init">
+        <oui-spinner></oui-spinner>
+     </div>
+
+    <div data-ng-if="!$ctrl.loading.init">
+        <div data-ovh-alert="billing_payment_request"></div>
+
+        <div data-ng-if="!$ctrl.requestPending">
+            <div class="mb-4">
```
Comment (antleblanc) [inline client/app/account/billing/payments/request/billing-payments-request.html]: You can apply your `data-ng-if="!$ctrl.requestPending"` here if you want:)

Comment (Jisay) [reply client/app/account/billing/payments/request/billing-payments-request.html]: Nope, cause datagrid is hidden during loading

File: client/app/account/billing/history/balance/billing-history-balance.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                    </span>
+                </oui-message>
+            </div>
+
+            <div class="row"
+                 data-ng-if="$ctrl.paymentMethods.length">
+                <div class="col-md-6">
+                    <form data-ng-submit="$ctrl.onBalancePayFormSubmit()">
+                        <div class="form-group">
+                            <label data-translate="billing_history_balance_choose_payment_method"></label>
```
[the remark this answers is not in the document]
Comment (Jisay) [reply client/app/account/billing/history/balance/billing-history-balance.html]: Shame on me :-$

**Pull request merged**